modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/metrics/Metrics.scala

```diff
@@ -234,4 +234,61 @@ object Metrics {
       }
   }
 
+  def noopIO: Metrics[IO] =
```

Comment:
if it's something used by tests only, it should go to the test runtime, not to the compile